api_definition.go

```diff
@@ -990,7 +990,7 @@ func (a *APISpec) CheckSpecMatchesStatus(r *http.Request, rxPaths []URLSpec, mod
 			}
 		case URLRewrite:
 			if r.Method == v.URLRewrite.Method {
-				return true, &v.URLRewrite
+				return true, v.URLRewrite
```

Comment:
Not sure if I follow, here you actually removed usage of pointer, what I'm missing?

Reply:
/cc @matiasinsaurralde

Reply:
`URLRewrite` was changed, so the `&` isn't required.
Without this patch this is the behavior:
```
Sending request for the first time:
meta.MatchRegexp = 0xc4204ada88
meta.MatchRegexp is nil, calling regexp.Compile
Sending the same request again:
meta.MatchRegexp = 0xc42049ea88
meta.MatchRegexp is nil, calling regexp.Compile
```
Using the patch:
```
Sending request for the first time:
meta.MatchRegexp = 0xc4203a8478
meta.MatchRegexp is nil, calling regexp.Compile
Sending the same request again:
meta.MatchRegexp = 0xc4203a8478
```